javascript/builtins/FinalizationRegistry.json

```diff
@@ -124,6 +136,10 @@
         "symbol_as_target": {
           "__compat": {
             "description": "Non-registered symbol as target",
+            "tags": [
+              "web-features:snapshot:ecmascript-2023",
```

Comment:
I see that https://262.ecma-international.org/14.0/#sec-finalization-registry-objects says "when target objects and symbols are garbage collected" and the "symbols" mention wasn't in https://262.ecma-international.org/13.0/#sec-finalization-registry-objects, so this seems like it's correct.